src/index.js

```diff
@@ -84,9 +84,9 @@ function createLoader() {
 function initializePal(loader) {
   let type;
 
-  const isElectronRenderer = isNodeLike && process.type === 'renderer';
+  var isRenderer = isNodeLike && (process.type === 'renderer' || process.versions['node-webkit']);
```

Comment:
If you could do one last change, and use `const` instead of `var` I'd really appreciate it :)
We try not to use `var` if possible.

Looks good otherwise!